### ISSUE TITLE
add scoped packages

### ISSUE THREAD

**Issue by rlidwka** [rlidwka/sinopia#104]
Would be in the next npm release, so we'll need to support that.

npm/npm#5239

**Comment by rlidwka** [comment]
@valmiRe and others who had that nginx issue: we added a workaround (https://github.com/rlidwka/sinopia/pull/275, https://github.com/rlidwka/sinopia/commit/fde2321222486c6cc5e200a39384ca3172c761ef). I hope it solves this.

**Comment by RnbWd** [comment]
Sorry I should have mentioned something months ago (maybe I did and don't remember). But I forked the repo and added one line of code that was relevant to the docker `nginx-proxy` package:

```js
  // for nginx proxy 
  if (process.env.VIRTUAL_HOST) app.enable('trust proxy')
```

in the `index.js` file - now docker just uses this repo here, which probably broke everyone's builds who were using my docker container with nginx. There were always solutions in the config file to handle some of the nginx stuff - but I apologize if I someone happened to be using my fork and the build broke.
